glmmTMB/vignettes/covstruct.rmd

```diff
@@ -208,11 +216,17 @@ Generate a dataset with 1000 groups:
 dat1 <- do.call("rbind", lapply(1:1000, simGroup) )
 ```
 
-And fitting the model on this larger dataset gives estimates close to
+Notice that we call `simGroup` 1000 times rather than drawing from a (1000*25)-variate normal distribution because observations beloging to different groups are uncorrelated with each other.
+
+Fitting the model on this larger dataset gives estimates closer to
 the true values (AR standard deviation=1, residual (measurement) standard deviation=1, autocorrelation=0.7):
 
 ```{r fit.ar1}
 (fit.ar1 <- glmmTMB(y ~ ar1(times + 0 | group), data=dat1))
+
```

Comment:
see notes above.

You could also show the confidence intervals for the correlation coefficient: for the previous (`n=25`) model, we get

```
confint(mod)["Cor.times2.times1|group",]
     2.5 %     97.5 %   Estimate 
-0.7036286  0.9239942  0.5805521 
```

i.e., this covers almost the entire [-1,1] range